astropy/coordinates/sites.py

```diff
@@ -124,23 +126,46 @@ def from_json(cls, jsondb: Mapping[str, dict[str, Any]]) -> Self:
 
 def get_builtin_sites() -> SiteRegistry:
     """
-    Load observatory database from data/observatories.json and parse them into
+    Load observatory database from ``data/observatories.json`` and parse them into
     a SiteRegistry.
     """
     jsondb = json.loads(get_pkg_data_contents("data/sites.json"))
     return SiteRegistry.from_json(jsondb)
 
 
-def get_downloaded_sites(jsonurl: str | None = None) -> SiteRegistry:
+def get_downloaded_sites(
+    jsonurl: str | None = None, cache: bool | str = True
```

Comment:
I don't think this module is exposed in API doc at all, so I consider this changing private API and no need change log mention.